src/test/java/org/opensearch/neuralsearch/query/HybridQuerySortIT.java

```diff
@@ -0,0 +1,705 @@
+    private static final String TEXT_FIELD_VALUE_2_DUNES = "Dunes part 2";
+    private static final String TEXT_FIELD_VALUE_3_MI_1 = "Mission Impossible 1";
+    private static final String TEXT_FIELD_VALUE_4_MI_2 = "Mission Impossible 2";
+    private static final String TEXT_FIELD_VALUE_5_TERMINAL = "The Terminal";
+    private static final String TEXT_FIELD_VALUE_6_AVENGERS = "Avengers";
+    private static final int INTEGER_FIELD_STOCK_1_25 = 25;
+    private static final int INTEGER_FIELD_STOCK_2_22 = 22;
+    private static final int INTEGER_FIELD_STOCK_3_256 = 256;
+    private static final int INTEGER_FIELD_STOCK_4_25 = 25;
+    private static final int INTEGER_FIELD_STOCK_5_20 = 20;
+    private static final String KEYWORD_FIELD_CATEGORY_1_DRAMA = "Drama";
+    private static final String KEYWORD_FIELD_CATEGORY_2_ACTION = "Action";
+    private static final String KEYWORD_FIELD_CATEGORY_3_SCI_FI = "Sci-fi";
+    private static final int SHARDS_COUNT_IN_SINGLE_NODE_CLUSTER = 1;
+    private static final int SHARDS_COUNT_IN_MULTI_NODE_CLUSTER = 3;
+    private static final int LTE_OF_RANGE_IN_HYBRID_QUERY = 400;
+    private static final int GTE_OF_RANGE_IN_HYBRID_QUERY = 20;
+    private static final int SMALLEST_STOCK_VALUE_IN_QUERY_RESULT = 20;
+    private static final int LARGEST_STOCK_VALUE_IN_QUERY_RESULT = 400;
+    private static final int LARGEST_STOCK_VALUE_IN_SEARCH_AFTER_MULTIPLE_FIELD_QUERY_RESULT = 25;
+    private static final int LARGEST_STOCK_VALUE_IN_SEARCH_AFTER_SINGLE_FIELD_QUERY_RESULT = 22;
+
+    @BeforeClass
+    @SneakyThrows
+    public static void setUpCluster() {
+        // we need new instance because we're calling non-static methods from static method.
+        // main purpose is to minimize network calls, initialization is only needed once
+        HybridQuerySortIT instance = new HybridQuerySortIT();
+        instance.initClient();
+        instance.updateClusterSettings();
+    }
+
+    @Override
+    protected boolean preserveClusterUponCompletion() {
+        return true;
+    }
+
+    // TODO Enable the test after catering the fix for https://github.com/opensearch-project/neural-search/issues/799
+    // @SneakyThrows
+    // public void testSortOnSingleShard_WhenConcurrentSearchEnabled_thenSuccessful() {
+    // try {
+    // updateClusterSettings("search.concurrent_segment_search.enabled", true);
+    // prepareResourcesBeforeTestExecution(SHARDS_COUNT_IN_SINGLE_NODE_CLUSTER);
+    // testSingleFieldSort_whenMultipleSubQueries_thenSuccessful(TEST_MULTI_DOC_INDEX_WITH_TEXT_AND_INT_SINGLE_SHARD);
+    // testMultipleFieldSort_whenMultipleSubQueries_thenSuccessful(TEST_MULTI_DOC_INDEX_WITH_TEXT_AND_INT_SINGLE_SHARD);
+    // } finally {
+    // wipeOfTestResources(TEST_MULTI_DOC_INDEX_WITH_TEXT_AND_INT_SINGLE_SHARD, null, null, SEARCH_PIPELINE);
+    // }
+    // }
+
+    @SneakyThrows
+    public void testSortOnSingleShard_WhenConcurrentSearchDisabled_thenSuccessful() {
```

Comment:
```suggestion
    public void testSortOnSingleShard_whenConcurrentSearchDisabled_thenSuccessful() {
```

please fix the case in other method names too